src/components/AddBook.js

```diff
@@ -13,12 +13,13 @@ function AddBook() {
   function addNewBook(event) {
     event.preventDefault();
     const newBook = {
-      id: uuidv4(),
+      item_id: uuidv4(),
       title,
       author,
+      category: 'Action',
     };
 
-    dispatch(addBook(newBook));
+    dispatch(saveBook(newBook));
```

Comment:
You have done great implementing API calls for the actions. Kindly ensure that the actions are dispatched as they should. Inspecting the network section of the chrome dev, there seems not to be an actual API call on adding a book. The same goes for removing a book. Kindly fix this.

![image](https://user-images.githubusercontent.com/20554583/204067634-2a1d70a6-6492-4914-84b3-067664417200.png)

Reply:
Hello @emmiiorji!! 👋🏾 
Thank you for the quick review. I actually made API requests on the delete and add buttons. I just repeated the actions of adding a book and removing one. And I can see the results on POSTMAN with the book added when I add and another one deleted whenever I click on the REMOVE button. And there is no way in the code to add the book to the page if it doesn't come from the API. Please check again